hw/top_earlgrey/data/ip/chip_clkmgr_testplan.hjson

```diff
@@ -0,0 +1,330 @@
+              "manuf_cp_unlock_raw",
+	      "manuf_cp_yield_test"]
+      bazel: [
+        "//sw/device/tests:ast_clk_outs_test_fpga_cw310_test_rom",
+	"//sw/device/tests:clkmgr_external_clk_src_for_sw_fast_test_fpga_cw310_test_rom",
+	"//sw/device/tests:clkmgr_external_clk_src_for_sw_slow_test_fpga_cw310_test_rom",
+      ]
+    }
+    {
+      name: chip_sw_clkmgr_external_clk_src_for_lc
+      desc: '''Verify the clkmgr requests ext clk src during certain LC states.
+
+            On POR lc asserts lc_clk_byp_req on some LC states, and de-asserts
+            it when lc_program completes. This also triggers divided clocks to step down. It may be
+            best to verify this via SVA, unless we implement clock cycle counters.
+	    SiVal: There is no way to test clock frequencies before clocks are calibrated.
+	    So the only way to actually test this is by the success of OTP updates for calibration.
+	    This can be tested after the fact by clock measurements of the internal, calibrated clocks.
+            '''
+      stage: V2
+      si_stage: SV1
+      tests: [
+        "chip_sw_clkmgr_external_clk_src_for_lc",
+	"manuf_cp_unlock_raw",
+      ]
+    }
+    {
+      name: chip_sw_clkmgr_external_clk_src_for_sw
+      desc: '''Verify SW causes the clkmgr requests ext clk src during certain LC states.
+
+            In RMA and TEST_UNLOCKED lc states the external clock is enabled in response to
+            `extclk_ctrl.sel` CSR writes. In addition `extclk_ctrl.hi_speed_sel` CSR causes the
+            divided clocks to step down. Verify this via SVA bound to clkmgr, and clock cycle
+            counters.
+
+            Disable external clock source and verify the AST reliably falls back to the internal
+            clock. Ensure the chip operates normally.
+            X-ref with chip_sw_uart_tx_rx_alt_clk_freq, which needs to deal with this as well.
+	    SiVal: The only necessary test is that the internal clocks are running
+	    with the right frequency after calibration.
+            '''
+      stage: V2
+      si_stage: None
+      tests: ["chip_sw_clkmgr_external_clk_src_for_sw_fast_test_unlocked0",
+              "chip_sw_clkmgr_external_clk_src_for_sw_slow_test_unlocked0",
+              "chip_sw_clkmgr_external_clk_src_for_sw_fast_dev",
+              "chip_sw_clkmgr_external_clk_src_for_sw_slow_dev",
+              "chip_sw_clkmgr_external_clk_src_for_sw_fast_rma",
+              "chip_sw_clkmgr_external_clk_src_for_sw_slow_rma"]
+    }
+    {
+      name: chip_sw_clkmgr_jitter
+      desc: '''Verify the clock jitter functionality.
+
+            Enable clock jitter setting the clkmgr `jitter_enable` CSR high. This causes the
+            jitter_o clkmgr output to toggle. Verify this output is connected to AST's
+            clk_src_sys_jen_i input using formal.
+
+            X-ref with various specific jitter enable tests.
+            '''
+      stage: V2
+      tests: ["chip_sw_clkmgr_jitter",
```

Comment:
you didn't put si_stage. Is this because it is not si_val test?

Reply:
Right. This is a connectivity test. The test below is the real SiVal test.